src/cc_catalog_airflow/dags/provider_api_scripts/metropolitan_museum_of_art.py

```diff
@@ -0,0 +1,279 @@
+"""
+Content Provider:       Metropolitan Museum of Art
+
+ETL Process:            Use the API to identify all CC0 artworks.
+
+Output:                 TSV file containing the image, their respective meta-data.
+
+Notes:                  https://metmuseum.github.io/
+                        No rate limit specified.
+"""
+
+import argparse
+import os
+import common.requester as requester
+import common.storage.image as image
+import logging
+from datetime import datetime, timedelta, timezone
+
+
+DELAY   = 1.0 #time delay (in seconds)
+FILE    = 'metmuseum_{}.tsv'.format(datetime.now())
+LIMIT = 500
+MEAN_GLOBAL_USAGE_LIMIT = 10000
```

Comment:
`MEAN_GLOBAL_USAGE_LIMIT` was supposed to be used by only `wikimedia_commons.py`. It is also not needed. I think you should remove it.